.gitignore

```diff
@@ -189,13 +189,13 @@ src/include/pbs_version.h
 src/include/pbs_config.h
 src/include/pbs_config.h.in
 src/include/pbs_config.h.in~
-src/server/queue_attr_def.c
-src/server/resc_def_all.c
-src/server/resv_attr_def.c
-src/server/sched_attr_def.c
-src/server/svr_attr_def.c
-src/server/job_attr_def.c
-src/server/node_attr_def.c
+src/lib/Libattr/queue_attr_def.c
+src/lib/Libattr/resc_def_all.c
+src/lib/Libattr/resv_attr_def.c
+src/lib/Libattr/sched_attr_def.c
+src/lib/Libattr/svr_attr_def.c
+src/lib/Libattr/job_attr_def.c
+src/lib/Libattr/node_attr_def.c
```

Comment:
My original thought was to move the is_attr() functionality into ecl since ecl had the attr_def files already linked in.  That way these stay with the server.  I guess they could move into libattr.  Having is_attr() in libattr makes sense.